uaclient/security_status.py

```diff
@@ -67,17 +74,15 @@ def filter_security_updates(
     Checks if the package has a greater version available, and if the origin of
     this version matches any of the series' security repositories.
     """
-    series = get_platform_info()["series"]
-    security_repos = [
-        template.format(series) for template in SECURITY_REPO_TEMPLATES
-    ]
-
     return [
         version
         for package in packages
         for version in package.versions
         if version > package.installed
-        and any(origin.archive in security_repos for origin in version.origins)
+        and any(
+            (origin.origin, origin.archive) in ORIGIN_INFORMATION_TO_SERVICE
```

Comment:
Why do we need to check both origin and archive here ? Wouldn't checking just one of them enough ?